server/lostandfound/urls.py

```diff
@@ -19,4 +19,5 @@
 admin.site.site_header = "Lost and Found Administration"
 urlpatterns = [
     url(r'^admin', admin.site.urls),
+    url(r'^admin/', admin.site.urls),
```

Comment:
So instead of having two routes to the admin page, you can just add a `APPEND_SLASH` setting to the settings.py as shown in the attached picture.
That takes care of any routes that the user may want to add a trailing slash.

<img width="533" alt="screen shot 2017-04-02 at 2 01 01 pm" src="https://cloud.githubusercontent.com/assets/25608335/24587350/e8891d46-17ac-11e7-8ce5-943c4310699d.png">